addons/survey/wizard/survey_invite.py

```diff
@@ -74,8 +74,8 @@ def _get_default_author(self):
 
     @api.depends('partner_ids', 'survey_id')
     def _compute_existing_partner_ids(self):
-        existing_answers = self.survey_id.user_input_ids
-        self.existing_partner_ids = existing_answers.mapped('partner_id') & self.partner_ids
+        self.existing_partner_ids = self.partner_ids.browse(
+            set(self.survey_id.user_input_ids.partner_id.ids) & set(self.partner_ids.ids))
```

Comment:
Do we need to browse the resulting set of ids though?

```suggestion
        self.existing_partner_ids = list(
            set(self.survey_id.user_input_ids.partner_id.ids) & set(self.partner_ids.ids))
```

Reply:
Huh, I never knew that was possible :D

Yeah, seems even better

Reply:
It works because the field setter is taking of that case. I didn't know either but it seems a bit odd to me to set a list of ids instead of a recordset. If you prefer both, we can go for it; I will update the code that way.